scopes/generator/generator/new.cmd.ts

```diff
@@ -28,18 +31,22 @@ export class NewCmd implements Command {
     [
       'a',
       'aspect <aspect-id>',
-      'aspect-id of the template. mandatory for non-core aspects. helpful for core aspects in case of a name collision',
+      "aspect-id of the template's owner, mandatory for non-core aspects. helpful for core aspects in case of a name collision",
```

Comment:
what is the reference for "owner" here?
owner is a term in bit.cloud, not bit.

Reply:
It means the aspect that registers the template - very open to a better wording.

But it's definitely not the aspect-id of the template itself...

Reply:
oh, i see. maybe just "id of the aspect that registered the template" is enough? or even "full name of the aspect that registered the template"?

Reply:
id sounds fine to me (if it was good enough before...)
if we had consistency/guidelines for when to use name/id/full name, then we could use it here, but at the moment we dont have that